site/src/AppRouter.tsx

```diff
@@ -254,6 +262,11 @@ export const AppRouter: FC = () => {
                 element={<ExternalAuthPage />}
               />
 
+              <Route
+                path="/oauth2-provider/authorize/:appId"
```

Comment:
Good question, GitHub has it under `/login/oauth2/authorize` so I think maybe we should put it there as well.

@Emyrk just in case you have any thoughts.

Reply:
We moved it!